Automattic/wp-calypso
# Pricing page i5: display proper CRM price

### Changes proposed in this Pull Request

This PR updates the price of Jetpack CRM in the pricing page.

### Testing instructions

- Download the PR or visit the Calypso live link
- Run Calypso or Jetpack cloud
- Select a self-hosted Jetpack site
- Visit the pricing page
- Check that the price of CRM is 17USD, for both monthly and selected term

### Screenshots

<img width="361" alt="Screen Shot 2020-11-17 at 5 51 18 PM" src="https://user-images.githubusercontent.com/1620183/99460349-b09f5100-28fd-11eb-9050-704f6522cb85.png">
